### DISCUSSION TITLE
giscus-discussion

### DISCUSSION THREAD

**Discussion by giscus** [intergrav/devins-badges#16] [giscus]
# devins-badges/

Beautifully designed badges for many services.

https://intergrav.github.io/devins-badges/

**Comment by intergrav** [comment]
hi

**Reply by Andre601** [reply]
Giscus integration... Nice.

Also, hi!

**Reply by osfanbuff63** [reply]
Hi! Smart to put giscus on this site!

**Comment by Deftu** [comment]
Very awesome badges

**Reply by intergrav** [reply]
Thanks!

**Comment by beans-squared** [comment]
![tenor](https://user-images.githubusercontent.com/65502593/196020378-708190b9-4ddf-4050-802c-521ea51e19fc.gif)

**Comment by ghost** [comment]
y u rickroll me

**Reply by intergrav** [reply]
lol

**Comment by CompeyDev** [comment]
very cool.

**Comment by sylv256** [comment]
OwO add a "Made by Sylveon" badge uwu

**Reply by sylv256** [reply]
4 years

**Comment by intergrav** [comment]
Devin's Badges v3 is out with fixed svg sizes, a new badge size, and more. Join the discord server for more information.

**Comment by ghost** [comment]
**sexy** lookin' badges!

**Comment by brawaru** [comment]
ديفين حقيقي جدًا بسبب هذه الشارات

**Comment by intergrav** [comment]
نعم براوارو أنا حقيقي جدًا

**Comment by splewsh** [comment]
These badges are amazing! Thanks for making them.

**Comment by JuniorWolfgamingDE** [comment]
Love the badges!! Thank you very much for making them available to everyone.

**Comment by cylenia** [comment]
boo!

**Comment by heytittino** [comment]
yo, these badges are fire! 🔥
